code/modules/mob/living/simple_animal/friendly/triceratops.dm

```diff
@@ -0,0 +1,47 @@
+/mob/living/simple_animal/triceratops
+	name = "Bismuth"
+	desc = "Acient, Reliable, Good at Pathfinding."
+	icon = 'icons/mob/pets.dmi'
+	icon_state = "bismuth"
+	icon_living = "bismuth"
+	icon_dead = "bis_dead"
+	speak_emote = list("grumbles")
+	emote_hear = list("grunts.","grumbles.")
+	emote_see = list("wags their tail.", "sniffs at the ground.")
+	speak_chance = 1
+	turns_per_move = 5
+	butcher_results = list(/obj/item/dice/d20 = 1)
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "rams"
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	gold_core_spawnable = FRIENDLY_SPAWN
+	melee_damage_lower = 18
+	melee_damage_upper = 18
+	health = 350
+	maxHealth = 350
+	speed = 5
+	glide_size = 2
+	can_be_held = FALSE
+	footstep_type = FOOTSTEP_MOB_SHOE
+
+/mob/living/simple_animal/triceratops/Initialize(mapload)
+	var/cap = CONFIG_GET(number/bismuthcap)
+	if (LAZYLEN(SSmobs.bismuth) > cap)
+		if(prob(30))
+			new /mob/living/simple_animal/triceratops(loc)
```

Comment:
Uh, huh, why?

Reply:
its a reference

Reply:
bonus bismuth (up to 6)

Reply:
what

Reply:
> what

It is a reference to the game Pathfinder: Wrath of the righteous where you are given a tool which allows you to summon 6 dinosaurs, all named bismuth who scale with you. The reason there is 6 max is because you can only have 6 party members

Reply:
shouldn't it be 5 max, since how are you gonna use an item without anyone in the party

Reply:
> shouldn't it be 5 max, since how are you gonna use an item without anyone in the party

no trust me